tools/dart/dart_autoroller.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python -u
```

Comment:
Why hardcode the path? Not that we build on FreeBSD, and yes this is super specific to our infra, but this would break people whose python is at e.g. `/usr/local/bin/python`

https://stackoverflow.com/questions/2429511/why-do-people-write-the-usr-bin-env-python-shebang-on-the-first-line-of-a-pyt

Reply:
I _think_ the existing string was breaking things with the '-u' option, but it seems fine now... I'll change it back.